index.html

```diff
@@ -139,5 +139,39 @@ <h2>Skills</h2>
           </ul>
     </section>
 </section>
+<!-------- Last Page -------->
+<section id="lastpage">
+  <nav>
+  </nav>
+  <header> 
+     <h1>Get started</h1>
+     <p id="paragraph">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+     <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] [REQUIRED] Good job so far to implement some changes from the previous reviewer but your form is not working cause you forget to generate your own action link instead of using `"https://formspree.io/f/{form_id}"` which is just an example. In order to have your own action link, you should o [formspree service](https://formspree.io/html), first register or signup and then create a new project and then a new form as described in the video I'm sharing with you, please fix it :+1: 
https://www.loom.com/share/7447923120c8413a9fd14e0f58e5ea03


> * [x]  Please @aliakhraisha wave, make sure that your form includes an `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
>   
>   * `action` value should be similar to `https://formspree.io/f/{form_id}`
>   * `method` value equals `POST`
>     In your `form` you should have something like this point_down
> 
> `<form action="https://formspree.io/f/{form_id}" method="post" >....</form>`
> 
> * [x]  First, go to [formspree](https://formspree.io/html) create an account and follow the instruction on how to set up an action.